src/llvm.cr

```diff
@@ -73,7 +73,7 @@ module LLVM
     LibLLVM.is_multithreaded != 0
   end
 
-  def self.default_target_triple : String
+  def self.host_target_triple : String
```

Comment:
I only meant `Config.host_target`.
IMO this should stay `LLVM.default_target_triple` because it' just wraps `LibLLVM.get_default_target_triple`.